src/internal/commonTypes.ts

```diff
@@ -14,4 +14,44 @@
  * limitations under the License.
  */
 
+import { SecurityTetraplet } from '@fluencelabs/avm';
+
+/**
+ * Peer ID's id as a base58 string (multihash/CIDv0).
+ */
 export type PeerIdB58 = string;
+
+/**
+ * Additional information about a service call
+ */
+export interface CallParams<ArgName extends string | null> {
+    /**
+     * The identifier of particle which triggered the call
+     */
+    particleId: string;
+
+    /**
+     * The peer id which created the particle
+     */
+    initPeerId: PeerIdB58;
+
+    /**
+     * Particle's timestamp when it was created
+     */
+    timeStamp: number;
```

Comment:
timestamp is a single word, no reason for camelCase

Reply:
fixed